### ISSUE TITLE
[bug] `<rh-accordion>` required attribute `aria-level` not present on `<rh-accordion-header>`

### ISSUE THREAD

**Issue by adamjohnson** [RedHat-UX/red-hat-design-system#2161]
### Describe the bug

`<rh-accordion-header>` should have the `aria-level` attribute but it does not.

### Screenshots

![Screenshot of rh-accordion-header HTML](https://github.com/user-attachments/assets/75313962-656a-45b3-ac97-780abd91ec8d)

### Steps to reproduce

1. Go to the [`<rh-accordion>` demo](https://ux.redhat.com/elements/accordion/demo/)
2. Inspect "Item 1" (or whatever the first accordion header is called)
3. Note the lack of an `aria-level` attribute on `<rh-accordion-header>`
4. Run an axe DevTools test to confirm

### Expected behaviour

`<rh-accordion-header>` should have an `aria-level` attribute [as is required](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-level#with_heading_role).

### Element HTML

```HTML

```

### Operating System (OS)

macOS

### Browser

Microsoft Edge

### Additional browser or assistive technology info

Interestingly, if you disable JS, an aria-level attribute is present.

**Comment by bennypowers** [comment]
Shouldn't this be handled by element internals?
